### PR TITLE
[Bug] Clear the teamId when delete the member

### LINKED ISSUE #2008: [Bug] The lastTeamId may be wrong when delete the member
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

The lastTeamId may be wrong when delete the member

### StreamPark Version

The latest dev branch

### Java Version

_No response_

### Flink Version

all

### Scala Version of Flink

all

### Error Exception

```log
all
```

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by 1996fanrui**
## What changes were proposed in this pull request

Issue Number: close #2008 

The lastTeamId may be wrong when delete the member

## Brief change log

Clear the teamId when delete the member

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

**Comment by 1996fanrui** [comment]
Hi @macksonmu @wolfboys, please help take a look in your free time, thanks.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java [hunk 1]
```diff
@@ -120,6 +120,8 @@ public interface UserService extends IService<User> {
 
     void setLatestTeam(Long teamId, Long userId);
 
+    void clearDeletedTeamId(Long userId, Long teamId);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java]: how about `unbindTeam(Long userId, Long teamId)` ?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MemberServiceImpl.java [hunk 2]
```diff
@@ -136,14 +136,17 @@ public void createMember(Member member) {
...
+            .orElseThrow(() -> new IllegalArgumentException(String.format("The member [id=%s] not found", memberArg.getId())));
         this.removeById(member);
+        userService.clearDeletedTeamId(member.getUserId(), member.getTeamId());
     }
 
     @Override
     public void updateMember(Member member) {
         Member oldMember = Optional.ofNullable(this.getById(member.getId()))
-            .orElseThrow(() -> new IllegalArgumentException(String.format("The mapping [id=%s] not found", member.getId())));
+            .orElseThrow(() -> new IllegalArgumentException(String.format("The member [id=%s] not found", member.getId())));
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MemberServiceImpl.java]: Maybe throw `ApiAlertException` is better

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java (hunk 1 above)
Comment (1996fanrui) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java]: Hi @wolfboys , thanks for your review. I addressed all comments~

**Review by wolfboys — DISMISSED** [review]
LGTM, thanks for your contribution

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -188,6 +188,16 @@ public void setLatestTeam(Long teamId, Long userId) {
         this.baseMapper.updateById(user);
     }
 
+    @Override
+    public void unbindTeam(Long userId, Long teamId) {
+        User user = getById(userId);
+        AssertUtils.checkArgument(user != null);
```
Comment (muchunjin) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java:192]: hi, fan rui, thank you very much for the hard work, when the user is deleted first, then an exception will be thrown when the member is deleted. Is it possible to add a judgment, that is, returns when user is null.

```
if (user == null || !teamId.equals(user.getTeamId())) {
     return;
}
```

Comment (muchunjin) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java:192]: Is it possible that when a person leaves, I may delete this user directly

Comment (muchunjin) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java:192]: When deleting a user, the member dependency is not judged

**Comment by wolfboys** [comment]
cc @macksonmu please double check in your free time, thanks.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java (hunk 3 above)
Comment (1996fanrui) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java:192]: Hi @macksonmu , thanks for your review.

Good catch, updated. The member mapping  will be deleted when delete user.

**Comment by muchunjin** [comment]
LGTM, thanks for your contribution

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**